swap_meet/vendor.py

```diff
@@ -1,2 +1,138 @@
+from swap_meet.item import Item
+from swap_meet.electronics import Electronics
+from swap_meet.decor import Decor
+from swap_meet.clothing import Clothing
+
 class Vendor:
-    pass
\ No newline at end of file
+    '''
+    object contains an attribute: inventory that is an empty list with an optional parameter
+    it also contains the methods: add, remove, get_by_category, swap_items, swap_first_item,
+    get_best_by_category
+    '''
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        '''
+        appends item parameter to inventory list
+        returns the last item in the inventory list
+        '''
+        self.inventory.append(item)
+        last_inventory_item = self.inventory[-1]
+
+        return last_inventory_item
```

Comment:
This `last_inventory_item` code works, but it's doing some extra work.  It is possible to just return `item` here.